sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -57,7 +57,7 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
     /**
      * @brief Maximum number of attempts to retry.
      */
-    int MaxRetries = 3;
+    int32_t MaxRetries = 3;
```

Comment:
@antkmsft wasn't this a breaking change? Friendly reminder to add changelog entries for such changes.

Reply:
Nevermind, this change is fine/non-breaking.